mscore/inspector/inspectorInstrchange.cpp

```diff
@@ -26,13 +31,30 @@ InspectorInstrumentChange::InspectorInstrumentChange(QWidget* parent)
       ic.setupUi(addWidget());
 
       const std::vector<InspectorItem> il = {
+            { Pid::SUB_STYLE,        0, ic.style,        ic.resetStyle        },
             { Pid::PLACEMENT,        0, ic.placement,    ic.resetPlacement    }
             };
       const std::vector<InspectorPanel> ppList = {
             { ic.title, ic.panel }
             };
       populatePlacement(ic.placement);
+      populateStyle(ic.style);
       mapSignals(il, ppList);
+      connect(ic.selectInstrument, SIGNAL(clicked()), SLOT(selectInstrumentClicked()));
+      }
+
+//---------------------------------------------------------
+//   selectInstrumentClicked
+//---------------------------------------------------------
+
+void InspectorInstrumentChange::selectInstrumentClicked()
+      {
+      InstrumentChange* i = static_cast<InstrumentChange*>(inspector->element());
```

Comment:
Is there a `toInstrumentChange()` available? If so, then better use that.

Reply:
Ope, yes, I'm sure it is. I think there's a big list of element types somewhere and the toX() is generated automatically. If there is renewed interest in this I will make that change.

Reply:
I'm interested ;-)

Reply:
Done